src/Trie.mo

```diff
@@ -129,44 +131,23 @@ module {
     }
   };
 
-  /// Checks the invariants of the trie structure, including the placement of keys at trie paths
-  public func isValid<K, V>(t : Trie<K, V>, enforceNormal : Bool) : Bool {
-    func rec(t : Trie<K, V>, bitpos : ?Hash.Hash, bits : Hash.Hash, mask : Hash.Hash) : Bool {
+  public func assertIsValid<K, V>(t : Trie<K, V>) {
```

Comment:
Since this is already public function, rename isValid to assertIsValid is a breaking change (found while running my own test suite). I'd just keep the old one...